packages/components/select/src/multi-select.tsx

```diff
@@ -109,6 +117,7 @@ export const MultiSelect = forwardRef<MultiSelectProps, "div">((props, ref) => {
   const [styles, mergedProps] = useComponentMultiStyle("MultiSelect", props)
   let {
     className,
+    clearable = true,
```

Comment:
```suggestion
    clearable
```

Reply:
You suggest removing the default value of `clearable`, but I set this to match `isClearable = true`. 
Should I remove the default value?

https://github.com/yamada-ui/yamada-ui/pull/3686/files#diff-4fd53e910fe635301af83fbb115c3187ab09e3f8ba4563622610118bec6e38ffR129

Reply:
@kaehehehe 
You suggest removing the default value of `clearable`, but I set this to match `isClearable = true`. 
Should I remove the default value?

https://github.com/yamada-ui/yamada-ui/blob/main/packages/components/select/src/multi-select.tsx#L120

Reply:
@chida09 
It could be beneficial to set `clearable = true` when removing deprecated props in `version 2.0`.

Reply:
@kaehehehe 
Since removing default values would change the behavior, wouldn't it be better not to remove them starting from the deprecation stage?

Reply:
I think it would be better to set clearable = true when working with version 2.0! @chida09

Reply:
Fixed.
https://github.com/yamada-ui/yamada-ui/pull/3686/commits/2689aac87c5703595a36189df9bcd8feb1d04cb7?w=1#diff-4fd53e910fe635301af83fbb115c3187ab09e3f8ba4563622610118bec6e38ffL120